src/management/index.js

```diff
@@ -947,6 +947,7 @@ utils.wrapPropertyMethod(ManagementClient, 'updateRule', 'rules.update');
  * @param   {Number}    [params.search_engine] The version of the search engine to use.
  * @param   {Number}    [params.per_page]      Number of results per page.
  * @param   {Number}    [params.page]          Page number, zero indexed.
+ * @param   {String}    [params.q]             User Search string to filter which users are returned.
```

Comment:
Probably worth mentioning in the docs that it follows Lucence query string syntax as documented at string Query in Lucene query string syntax. Not all metadata fields are searchable when using search_engine=v2, for details see Searchable Fields (v2). When using search_engine=v3, some query types cannot be used on metadata fields, for details see Searchable Fields (v3)

Reply:
Yeah good spot, add that and an example now!